src/Raven.Client/ServerWide/Operations/Logs/PutTrafficWatchConfigurationOperation.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using Raven.Client.Documents.Changes;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Http;
+using Raven.Client.Json;
+using Sparrow.Json;
+using Size = Sparrow.Size;
+
+namespace Raven.Client.ServerWide.Operations.Logs
+{
+    internal class PutTrafficWatchConfigurationOperation : IServerOperation
+    {
+        private readonly Parameters _parameters;
+
+        public class Parameters
```

Comment:
You can remove this and use the `TrafficWatchConfigurationResult` instead

Reply:
Done.